docker/web/Dockerfile

```diff
@@ -0,0 +1,125 @@
+###############################################################################
+#-----------------------------    BUILD STAGE   ------------------------------#
+###############################################################################
+
+FROM ubuntu:bionic as builder
+
+ARG CC_VERSION=master
+ENV CC_VERSION ${CC_VERSION}
+
+ARG CC_DATABASE=sqlite
+ENV CC_DATABASE ${CC_DATABASE}
+
+ARG CC_BUILD_TYPE=Release
+ENV CC_BUILD_TYPE ${CC_BUILD_TYPE}
+
+RUN set -x && apt-get update -qq \
+  && apt-get install -qqy \
+    cmake make \
+    llvm-7 clang-7 llvm-7-dev libclang-7-dev \
+    libboost-all-dev \
+    odb libodb-sqlite-dev libodb-pgsql-dev \
+    libsqlite3-dev \
+    default-jdk ant\
+    libssl1.0-dev \
+    libgraphviz-dev \
+    libmagic-dev \
+    libgit2-dev \
+    nodejs-dev node-gyp npm \
+    ctags \
+    wget \
+    git \
+    libgtest-dev \
+    && \
+    # Build thrift.
+    wget "http://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=thrift/0.12.0/thrift-0.12.0.tar.gz" -O /opt/thrift-0.12.0.tar.gz; \
+    tar -xf /opt/thrift-0.12.0.tar.gz -C /opt; \
+    cd /opt/thrift-0.12.0; \
+    ./configure --prefix=/opt/thrift --silent; \
+    cd /; \
+    make -C /opt/thrift-0.12.0 -j4 --silent install; \
+    rm -rf /opt/thrift-0.12.0 /opt/thrift-0.12.0.tar.gz; \
+    # Build GTest.
+    cd /usr/src/googletest/ && \
+    mkdir build && \
+    cd build && \
+    cmake .. && \
+    make install && \
+    cd / && \
+    rm -rf /usr/src/googletest/build
+
+ENV PATH="/opt/thrift/bin:$PATH"
+ENV CMAKE_PREFIX_PATH="/opt/thrift:$CMAKE_PREFIX_PATH"
+
+# Download CodeCompass release.
+RUN git clone https://github.com/Ericsson/CodeCompass.git /CodeCompass
+WORKDIR /CodeCompass
+RUN git checkout ${CC_VERSION}
+
+# Build CodeCompass.
+RUN mkdir /CodeCompass-build && \
+  cd /CodeCompass-build && \
+  cmake /CodeCompass \
+    -DDATABASE=$CC_DATABASE \
+    -DCMAKE_INSTALL_PREFIX=/CodeCompass-install \
+    -DCMAKE_BUILD_TYPE=$CC_BUILD_TYPE && \
+  make && \
```

Comment:
This `make` fails with the following error message:
```
[  7%] Generating ODB for include/model/statistics.h
cc1plus: error: bad value ('/dev/null') for '-mtune=' switch
cc1plus: note: valid arguments to '-mtune=' switch are: nocona core2 nehalem corei7 westmere sandybridge corei7-avx ivybridge core-avx-i haswell core-avx2 broadwell skylake skylake-avx512 bonnell atom silvermont slm knl intel x86-64 eden-x2 nano nano-1000 nano-2000 nano-3000 nano-x2 eden-x4 nano-x4 k8 k8-sse3 opteron opteron-sse3 athlon64 athlon64-sse3 athlon-fx amdfam10 barcelona bdver1 bdver2 bdver3 bdver4 znver1 btver1 btver2 generic
make[2]: *** [model/statistics-odb.cxx] Error 1
model/CMakeFiles/model.dir/build.make:166: recipe for target 'model/statistics-odb.cxx' failed
make[1]: *** [model/CMakeFiles/model.dir/all] Error 2
CMakeFiles/Makefile2:179: recipe for target 'model/CMakeFiles/model.dir/all' failed
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

Reply:
See #384